backend/django/core/views/frontend.py

```diff
@@ -200,6 +192,35 @@ def get_form_prefix(self, step=None, form=None):
 
         return prefix
 
+    def get_context_data(self, form, step=None, **kwargs):
+        context = super(ProjectCreateWizard, self).get_context_data(form=form, **kwargs)
+
+        if self.steps.current == "project":
+            user = self.request.user.profile
+
+            if str(user) == "root":
```

Comment:
Quick question: why does this user get their own case? I think the root user is just a sample user to test SMART with.